test-requirements.txt

```diff
@@ -1,18 +1,7 @@
-# This file is managed centrally by release-tools and should not be modified
```

Comment:
Same here, you have to respect this disclaimer. Don't turn this charm into a snowflake please :)